docs/source/user_manual/tutorial_van_der_waal.rst

```diff
@@ -285,24 +320,26 @@ Source Code
 The final version on the program, `vanderwaals.py` ::
```

Comment:
Let's not explicitly copy the code from the python file here - instead just use `literalinclude`. Also, it would be awesome if we provide a way to click on `vanderwaals.py` and see the example on github. I think that's possible - not sure how.

Reply:
I've tried doing both of these things. The link to github is tricky because the file doesn't exist yet, it is bing introduced in this PR.  The link I used is where the file _will_ be located, but currently it is a broken link if you try to test it on this PR... 
I am not sure the best way to go about this